config/application.rb

```diff
@@ -11,5 +11,6 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    config.middleware.use Rack::Deflater
```

Comment:
this is for compressing the json? wouldn't the nginx in the docker image perform this?